app/InteractiveOptions/Path.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\InteractiveOptions;
+
+use App\Commands\NewCommand;
+use Illuminate\Support\Facades\File;
+use App\Support\BaseInteractiveOption;
+
+class Path extends BaseInteractiveOption
+{
+    /**
+     * Option key.
+     *
+     * @var string
+     */
+    protected $key = 'path';
+
+    /**
+     * Performs the option interactively.
+     *
+     * @param NewCommand $console
+     * @return BaseInteractiveOption
+     */
+    public function perform(NewCommand $console): BaseInteractiveOption
+    {
+        $question = 'New path for installation? (empty for current working dir)';
+
+        $answer = $console->ask($question, 'cwd');
+
+        if ($answer === 'cwd') {
+            $this->value        = $console->currentWorkingDir;
+        } elseif (!$this->pathIsDirectory($answer, $console->currentWorkingDir)) {
+            $this->value        = null;
+            $this->message      = "Provided path [{$answer}] is not a directory. Path config value wasn't changed.";
+            $this->messageLevel = 'error';
+        }
+
+        return $this;
+    }
+
+    /**
+     * Checks if the given path is a directory.
+     *
+     * @param $path
+     * @param $cwd
+     * @return bool
+     */
+    protected function pathIsDirectory($path, $cwd): bool
+    {
+        if (starts_with($path, '~')) {
```

Comment:
If we do end up sticking with this parsing, let's make something global--a helper or library--for converting these paths so we don't have the code replicated everywhere.